apps/front/package.json

```
@@ -18,31 +18,46 @@
 		"db:studio": "drizzle-kit studio"
 	},
 	"devDependencies": {
+		"@eslint/compat": "^1.2.8",
```

Biome (1.9.4)

[error] The key @eslint/compat was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)